packages/block-library/src/image/edit.js

```diff
@@ -430,29 +427,19 @@ export class ImageEdit extends Component {
 				<>
 					{ controls }
 					<Block.div
-						className={ classnames( className, {
-							[ `align${ align }` ]:
-								! needsAlignmentWrapper && align,
-						} ) }
+						className={ classnames( className, `align${ align }` ) }
 					>
-						{ needsAlignmentWrapper ? (
-							<div className={ `align${ align }` }>
-								{ mediaPlaceholder }
-							</div>
-						) : (
-							mediaPlaceholder
-						) }
+						{ mediaPlaceholder }
 					</Block.div>
 				</>
 			);
 		}
 
-		const classes = classnames( className, {
+		const classes = classnames( className, `align${ align }`, {
```

Comment:
I believe if we do this change we need to switch the classNames at the same time because of BC

Reply:
I changed the classes.